tests/testthat/test-check_tbl_value_col_ascending.R

```diff
@@ -70,3 +70,14 @@ test_that("check_tbl_value_col_ascending skips correctly", {
     check_tbl_value_col_ascending(tbl, file_path)
   )
 })
+
+
+test_that("(#78) check_tbl_value_col_ascending handle cdf char values", {
+
+  file_path <- test_path("testdata/files/2024-08-12-cdf-ascent.csv")
+  ex <- arrow::read_csv_arrow(file_path)
+
+  res <- hubValidations:::check_tbl_value_col_ascending(ex, file_path = "")
```

Comment:
oops no need to include pkg name here

```suggestion
  res <- check_tbl_value_col_ascending(ex, file_path = "")
```